packages/babel-preset-env/src/polyfills/corejs3/built-in-definitions.js

```diff
@@ -358,7 +363,7 @@ export const StaticProperties: ObjectMap<ObjectMap<string | string[]>> = {
       "esnext.promise.all-settled",
       ...PromiseDependenciesWithIterators,
     ],
-    any: ["esnext.promise.any", ...PromiseDependenciesWithIterators],
+    any: ["esnext.promise.any", ...PromiseAnyDependencies],
```

Comment:
I think that it would be better just add `esnext.aggregate-error` to this line since at this moment it's used only here and I have no ideas where`PromiseAnyDependencies` can be used in the future.

Reply:
Can do. I thought it was some unwritten code convention that almost every entry is written as `[name, ...depedencies]`.